src/components/QuranReader/SidebarNavigation/SidebarNavigation.tsx

```diff
@@ -70,10 +70,14 @@ const SidebarNavigation = () => {
     <div
       ref={sidebarRef}
       className={classNames(styles.container, {
-        [styles.visibleContainer]: isVisible === true,
-        [styles.containerAuto]: isVisible === 'auto',
-        [styles.spaceOnTopMenuExpanded]: isContextMenuExpanded,
-        [styles.spaceOnTopMenuCollapse]: !isContextMenuExpanded,
+        [styles.visibleContainer]: isSidebarVisible === true && isNavbarVisible,
+        [styles.visibleContainerCollapsed]: isSidebarVisible === true && !isNavbarVisible,
+        [styles.containerAuto]: isSidebarVisible === 'auto' && isNavbarVisible,
+        [styles.containerAutoCollapsed]: isSidebarVisible === 'auto' && !isNavbarVisible,
+        [styles.inVisibleContainer]: isNavbarVisible,
+        [styles.inVisibleContainerCollapsed]: !isNavbarVisible,
+        [styles.spaceOnTop]: isSidebarVisible && isNavbarVisible,
+        [styles.spaceOnTopCollapsed]: isSidebarVisible && !isNavbarVisible,
```

Comment:
I see there are a few duplications in this section. For example:
- `isSidebarVisible === true && isNavbarVisible` and `isSidebarVisible && isNavbarVisible`
- `isSidebarVisible && !isNavbarVisible` and `isSidebarVisible === true && !isNavbarVisible`

We should keep it DRY and technically they should share the same exact classes

Reply:
`isSidebarVisible === true` is different from `isNavbarVisible` as `isNavbarVisible` could be `true` or `auto`, however the first condition applies only when `isNavbarVisible` equals to `true` only, same for the second point.

Reply:
If this is the case, the code is very hard to follow. We defo need to refactor it with better naming and introduce variables. E.g. extract `isSidebarVisible === 'auto'` to a variable since it's used twice for example.

Reply:
`isSidebarVisible === true` is different from `isSidebarVisible` as `isSidebarVisible` could be `true` or `auto`.